editor/test/integration/test_util.clj

```diff
@@ -592,9 +592,9 @@
   directory path must be a temp directory."
   ^java.lang.AutoCloseable [directory-path]
   (let [directory (io/file directory-path)]
-    (assert (string/starts-with? (.getAbsolutePath directory)
+    (assert (string/starts-with? (.getCanonicalPath directory)
```

Comment:
This assert would fail for some tests on macOS after I made the project path canonical. Not sure why I didn't notice it before. 😞